docs/tethys_sdk/gizmos/bokeh_view.rst

```diff
@@ -2,7 +2,24 @@
 Bokeh View
 **********
 
-**Last Updated:** November 11, 2016
+**Last Updated:** August 2023
+
+.. important::
+
+   This gizmo requires the `bokeh` library to be installed. If you are using `microtethys`, you will need to install `bokeh` using conda or pip as follows:
+
+   .. code-block:: bash
+
+      # conda: conda-forge channel strongly recommended
+      conda install -c conda-forge "bokeh<3"
+
+      # pip
+      pip install "bokeh<3"
+
+   **Don't Forget**: If you end up using this gizmo in your app, add `bokeh` as a requirement to your `install.yml`.
```

Comment:
@sdc50 I took a first pass at a dependency install tip box in docs for optional dependencies. What do you think of this?

Reply:
Looks like this:

![image](https://github.com/tethysplatform/tethys/assets/5123221/4940c11b-49a9-4081-909f-fcca2d116ed2)

Reply:
This looks great! 

As a side note I think we can take off the pin for Bokeh. I've tested it with v3 and it still works great.

Reply:
One other thought, do we want to mention anything about Tethys v5.0?

Reply:
"This gizmo requires the `bokeh` library to be installed. Starting with Tethys 5.0 or if you are using `microtethys`, you will need to install `bokeh` using conda or pip as follows:"

Reply:
> As a side note I think we can take off the pin for Bokeh. I've tested it with v3 and it still works great.

I get an error in the showcase app when I have Bokeh 3 installed. See: #980